packages/cli/src/lingui-extract.ts

```diff
@@ -127,7 +127,7 @@ if (require.main === module) {
     hasErrors = true
     const msg =
       "--babelOptions option is deprecated." +
-      " Please set extractBabelOptions in configuration https://lingui.js.org/ref/conf.html#extractBabelOptions"
+      " Please set extractBabelOptions in configuration https://lingui.dev/ref/conf#extractBabelOptions"
```

Comment:
This anchor doesn't work as anchors are case sensitive. Please update `#extractBabelOptions` to be in lower-case